scaletest/terraform/coder.tf

```diff
@@ -0,0 +1,265 @@
+data "google_client_config" "default" {}
+
+locals {
+  coder_helm_repo    = "https://helm.coder.com/v2"
+  coder_helm_chart   = "coder"
+  coder_release_name = "coder-${var.name}"
```

Comment:
I see the value of including `var.name` in the Google resources, since there is a search you can do, but here I think it just makes things more wordy --- the idea is to have only one Coder instance per load-balancing cluster.

Reply:
Fair, and the namespace also includes `var.name`.